mixer/pkg/config/crd/store_test.go

```diff
@@ -385,16 +385,16 @@ func TestCriticalCrdsRetryMakeSucceed(t *testing.T) {
 			},
 		},
 	}
-	callCount := 0
+	var callCount int32
 	// Gradually increase the number of API resources.
 	fakeDiscovery.AddReactor("get", "resource", func(k8stesting.Action) (bool, runtime.Object, error) {
-		callCount++
-		if callCount == 2 {
+		atomic.AddInt32(&callCount, 1)
```

Comment:
should it catch the return value and use it below instead of using LoadInt32?

The value loaded may not be the same as the added value in this iteration.

Reply:
Similar in other places. should we load it atomically and store in a temp variable in the scope where the int is used multiple times?

Reply:
Thanks